conf/pipeline/raredisease/hasta.config

```diff
@@ -1,10 +1,25 @@
 process {
-    
+
     withName:'PICARD_MARKDUPLICATES' {
         memory = { check_max( 90.GB  * task.attempt, 'memory' ) }
     }
     withName:'DEEPVARIANT' {
         cpus   = { check_max( 16      * task.attempt, 'cpus' ) }
         memory = { check_max( 80.GB   * task.attempt, 'memory' ) }
     }
-}
\ No newline at end of file
+    withLabel:'sentieon' {
+        beforeScript = { "export PATH=\$PATH:\$SENTIEON_INSTALL_DIR/sentieon-genomics-202010.02/bin" }
+    }
+    withName: BCFTOOLS_VIEW {
+        if (params.genome == 'GRCh37') {
+            ext.args = '--output-type z --apply-filters PASS --exclude "INFO/clinical_genomics_mipAF > 0.40 | INFO/swegenAF > 0.40 | INFO/clingen_ngiAF > 0.40 | INFO/gnomad_svAF > 0.40 "'
```

Comment:
What do you think about listing the args as a list instead? Like what's done [here](https://github.com/nf-core/rnaseq/blob/89bf536ce4faa98b4d50a8ec0a0343780bc62e0a/conf/modules.config#L500)